plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/global-messages_en.properties

```diff
@@ -12,10 +14,10 @@ jpseo.label.config=Seo configuration
 jpseo.label.example=Eg
 
 jpseo.label.friendlyCode=Friendly code
-jpseo.label.pageDescription=Description
+jpseo.label.pageDescription=SEO Description
 jpseo.label.useBetterDescriptions=When available, use extra description in search.
-jpseo.label.xmlConfig=Complex parameters
-xmlConfig=Complex parameters
+jpseo.label.inheritFromDefaultLang=Inherit from default lang
+jpseo.label.addMetatags=Add Metatag
```

Comment:
What does this param jpseo.label.addMetatags do?

Reply:
"jpseo.label.addMetatags" the key of the label of the "Add metatag" section in the page edit UI